examples/updatecli.generic/yaml-files.yaml

```diff
@@ -0,0 +1,98 @@
+---
```

Comment:
For information the directory `examples` is deprecated in favor of `e2e/updatecli.d/success/` if the manifest doesn't contain warning or error message, `e2e/updatecli.d/deprecated` if you want to test deprecated message